frameworks/elastic/tests/test_sanity.py

```diff
@@ -76,6 +76,8 @@ def test_pod_replace_then_immediate_config_update():
 
     # ensure all nodes, especially data-0, get launched with the updated config
     config.check_plugin_installed(plugin_name, service_name=foldered_name)
+    sdk_plan.wait_for_completed_deployment(foldered_name)
```

Comment:
Should this not also wait for an in-progress deployment? (This was required for the toggle tests here: https://github.com/mesosphere/dcos-commons/blob/master/frameworks/kafka/tests/test_toggle_security.py#L535)

Reply:
No.  The Marathon app has been updated.  Assuming the update isn't totally broken it will never accidentally be complete.  In any case it never checked before.  It just fell through.  If it becomes an actual problem we can fix in a different PR.